WooCommerce/Classes/ViewRelated/Orders/Order Creation/OrderForm.swift

```diff
@@ -401,8 +411,23 @@ private struct ProductsSection: View {
                     Image(uiImage: .lockImage)
                         .foregroundColor(Color(.brand))
                         .renderedIf(viewModel.shouldShowNonEditableIndicators)
+
+                    HStack(spacing: OrderForm.Layout.productsHeaderButtonsSpacing) {
+                        scanProductButton
+                        .renderedIf(viewModel.isAddProductToOrderViaSKUScannerEnabled)
+
+                        Button(action: {
+                            showAddProduct.toggle()
+                        }) {
+                            Image(uiImage: .plusImage)
+                        }
+                        .id(addProductButton)
+                        .accessibilityIdentifier(OrderForm.Accessibility.addProductButtonIdentifier)
+                    }
+                    .scaledToFit()
+                    .renderedIf(!viewModel.shouldShowNonEditableIndicators && viewModel.shouldShowCustomAmountsWithProducts)
                 }
-                .renderedIf(!viewModel.shouldShowCustomAmountsWithProducts)
+                .renderedIf(!viewModel.shouldShowCustomAmountsWithProducts || viewModel.productRows.isNotEmpty)
```

Comment:
What do you think about wrapping these into `shoulRenderXYZ` properties in the viewModel?

Reply:
Good idea, done in 8828e127417ac49e022c5d321bdf76882367e1c3